### DISCUSSION TITLE
Dart support

### DISCUSSION THREAD

**Discussion by arnaudelub** [supabase/supabase#396] [Questions] [answered]
If you need any help for Dart support (101 thumbs up for Dart vs 93 for python) i'll be glad to help just in case you needed it :)
This is really a nice project!
Congrats for your last funding round, good luck and happy coding!

**Comment by kiwicopple** [comment, marked as answer]
@phamhieu has been building the Dart libraries: 

- https://github.com/supabase/postgrest-dart
- https://github.com/supabase/realtime-dart

Hieu - do you need help with the GoTrue library?

**Comment by phamhieu** [comment]
Thanks @arnaudelub, a Dart support for [GoTrue](https://github.com/supabase/gotrue-js) will be fantastic.
